config.yaml

```diff
@@ -14,7 +14,7 @@ carpentry: 'incubator'
 title: 'RNA-seq analysis with Bioconductor'
 
 # Date the lesson was created (YYYY-MM-DD, this is empty by default)
-created: '2020-09-15'
+created: ~
```

Comment:
I would not remove this. It provides metadata for the date the lesson was created (I want to find a good cross-platform solution for CI integration so that this can be left blank, but that's for the future). 

Was this done on purpose or was this done via a sandpaper function?

Reply:
This means it's a {sandpaper} issue. I suspect that it was an `add_episode_rmd()` or `move_episode()` situation. I will see if I can find a fix for it.